web/src/client/storage.js

```diff
@@ -117,25 +118,33 @@ class DevicesManager {
    * @property {string} [transport]
    * @property {boolean} [sdCard]
    * @property {boolean} [dellBOOS]
-   * @property {string[]} [devices] - RAID devices (only for "raid" type)
+   * @property {string[]} [devices] - RAID devices (only for "raid" and "md" types)
```

Comment:
I would not call it `RAID devices` and maybe instead `member devices` ?

Reply:
I have avoided `member` term because libstorage-ng uses it for a kind of BIOS RAID, see https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2storage/md_member.rb#L35.